### DISCUSSION TITLE
[A-L] (2023/24) Lezione 8 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#174] [Esercizi - Viaggi/Piazza]
Dimostrare la seguente proprietà sull'anello $(\mathbb{Z},+,\cdot)$:
- $MCD(a, \pm\ a)= |a|$

**Comment by Elia-Belli** [comment]
- Per definizione di $MCD(a,b)$, $(a,b)\neq (0,0)$, in questo caso abbiamo $(a,a)\Rightarrow a \neq 0 \Rightarrow a\geq 1$
- Mostro che $|a|$ divide sia $a$ che $-a$:
 Sia $a>0,\ (|a|)|a$, ovvero $a = |a|c$, con $c=1$
 Sia $a<0,\ (|a|)|a$, ovvero $a = |a|c$, con $c=-1$
- Dimostro $MCD(a,a)=|a|$ :
 Posso scrivere $(|a|)= ax_0+ay_0$ scegliendo $x_0=1,y_0=0$ se $a>0$, e $x_0=-1,y_0=0$ se $a<0$
 Se $d'|a \Rightarrow d'|ax_0+ay_0 = |a|$.
 Dimostro (analogamente) $MCD(a,-a)=|a|$ :
 Posso scrivere $(|a|)= ax_0-ay_0$ scegliendo $x_0=1,y_0=0$ se $a>0$, e $x_0=-1,y_0=0$ se $a<0$
 Se $d'|a\ \land\ d'|(-a) \Rightarrow d'|ax-ay,\ \forall x,\forall y \Rightarrow d'|ax_0-ay_0 = |a|$.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/268205dd-c3d0-4cb8-82ff-c6cb7af4e0d9)
